packages/block-editor/src/components/block-list/index.native.js

```diff
@@ -140,7 +139,7 @@ export class BlockList extends Component {
 						! isRootList && styles.overflowVisible,
 					] }
 					horizontal={ horizontal }
-					scrollEnabled={ scrollEnabled }
+					scrollEnabled={ isRootList }
```

Comment:
This one should lock scrolling of nested `InnerBlocks`. However I would like to make sure if it do not block something that we don't want to block.

The reason to introduce that prop lays in Column block PR . In nested structure there was possibility to drag content of Column or Group block.

Reply:
if you comment this line completely you will notice the behaviour presented on GIF in the description